CHANGELOG.md

```diff
@@ -6,6 +6,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.5.0] - Sep, 22, 2021
+
+### Core
+- [TextInput] Added Info message to inform user of dynamic input value changes
+- [Icons] New icons:
+  - Shopping cart
+
+### React Components
+
+#### Added
```

Comment:
Should we add the form validation additions to react components?

Reply:
?

Reply:
Now I get it: 
- [ ] Fix react changelog